### ISSUE TITLE
gpu branch and get_jk indexing

### ISSUE THREAD

**Issue by MatthewRHermes** [MatthewRHermes/mrh#61]
@cjknight 

https://github.com/MatthewRHermes/mrh/blob/99834c5941c878a94320092343360e8c8be29933/my_pyscf/mcscf/lasscf_async/lasscf_async.py#L57-L60

1. Style issue: can we have impure Python functions end with an underscore? (PySCF convention)

https://github.com/MatthewRHermes/mrh/blob/99834c5941c878a94320092343360e8c8be29933/gpu/gpu4pyscf/df/df_jk.py#L61

2. Would the "mode" business above become unnecessary if this were instead `numpy.zeros (dm_shape)`? Check this out:

```
>>> import numpy as np
>>> a = np.random.rand (4,5,5)
>>> b = a.transpose (1,0,2)
>>> a.strides
(200, 40, 8) # row-major order
>>> b.strides
(40, 200, 8) # neither row- nor column-major order
>>> c = np.zeros_like (b)
>>> d = np.zeros (b.shape)
>>> c.strides
(40, 200, 8) # same messy order as b
>>> d.strides
(160, 40, 8) # row-major order
```

**Comment by cjknight** [comment]
1. Yup, sure thing.
2. Ooooh, I'll take a look today. I really don't like that mode business. Replacing just the following with `vk = np.zeros(dms.shape)`seems too easy...

https://github.com/MatthewRHermes/mrh/blob/99834c5941c878a94320092343360e8c8be29933/gpu/gpu4pyscf/df/df_jk.py#L61

**Comment by MatthewRHermes** [comment]
Resolved by PR #67
